lib/data-serializer.js

```diff
@@ -18,11 +18,18 @@ class DataSerializer {
   }
 
   /**
-   * Get latest split and segment data and serialize into strings.
+   * Get latest split and segment data and return a scriprt of serialized strings.
    */
-  getSerializedData () {
-    // TODO: read from poller local cache and serialize data
-    throw new Error('Not implemented')
+  generateSerializedDataScript () {
+    const latestData = this.poller.cache
+    return `<script>
+      window.__splitCachePreload = {
```

Comment:
keys to match up with the keys the FE data loader script is using: https://github.com/godaddy/split-javascript-data-loader/pull/5/files#diff-be1a75c93d34bbecbd95759cc8ba0329R6